lib/vmpooler/providers/gce.rb

```diff
@@ -469,6 +490,9 @@ def purge_unconfigured_resources(allowlist)
 
               debug_logger("trigger async delete_instance #{vm.name}")
               result = connection.delete_instance(project, zone, vm.name)
+              vm_pool = vm.labels&.key?('pool') ? vm.labels['pool'] : nil
```

Comment:
I know this format is common, but I hate reading it. Not asking for a change, just needed to rant.